<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImplementingStandardSelectDlg</class>
 <widget class="QDialog" name="ImplementingStandardSelectDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>表号</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>限值类别：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="classNumBox"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>标准名称</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="tableNameBox"/>
   </item>
   <item row="10" column="1">
    <widget class="QPushButton" name="OkBtn">
     <property name="text">
      <string>确认</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>全选</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="MyComboBox" name="standardNameBox"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header>mycombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
